_automating-configurations/default-use-cases.md

```diff
@@ -0,0 +1,200 @@
+---
+layout: default
+title: Default Use Cases
+nav_order: 10
+---
+
+# Default Use Cases
+
+As part of our plugin we offer out of the box templates for some common use cases in the ML space. Our first set of pre-defined templates each execute a set of APIs to configure the starting point for popular ML use cases like semantic and conversational search. 
+
+While each default use case template has a defined schema and set of APIs with pre-defined defaults for each step, users can over-write any of these defaults as they choose too. The default use cases are added as part of our create API, meaning creating one of the default use cases doesn’t automatically provision them, however the optional provision param can be added to the request to create and provision these templates together. 
+
+
+## Use Case Tutorial Example:
+
+* One of the use cases we offer is semantic_search_with_cohere_embedding_query_enricher 
+* This predefined use case template will deploy a remote model, create an ingest pipeline with the new model, creates a sample KNN index and sets up a search pipeline to define the default model ID for that index.
+* Only field required here is the API key for cohere
+
+
+```
+POST /_plugins/_flow_framework/workflow?use_case=semantic_search_with_cohere_embedding_query_enricher&provision=true
+{
+    "create_connector.credential.key" : "<YOUR API KEY>"
+}
+```
+
+
+### You are ready to ingest and search now!
+
+```
+PUT /my-nlp-index/_doc/1
+{
+  "passage_text": "Hello world",
+  "id": "s1"
+}
+GET /my-nlp-index/_search
+{
+  "_source": {
+    "excludes": [
+      "passage_embedding"
+    ]
+  },
+  "query": {
+    "neural": {
+      "passage_embedding": {
+        "query_text": "Hi world",
+        "k": 100
+      }
+    }
+  }
+}
+```
```

Comment:
This can be split into multiple json blobs. Also a one-line description of what it's doing can be more informative.